server/src/main/java/io/druid/server/coordinator/rules/IntervalLoadRule.java

```diff
@@ -41,9 +42,9 @@ public IntervalLoadRule(
       @JsonProperty("tieredReplicants") Map<String, Integer> tieredReplicants
   )
   {
-    validateTieredReplicants(tieredReplicants);
+    this.tieredReplicants = tieredReplicants == null ? ImmutableMap.<String, Integer>of() : tieredReplicants;
```

Comment:
I wonder if the proper default for tiers should be `DruidServer.DEFAULT_TIER` and `DruidServer.DEFAULT_NUM_REPLICANTS`. That would be more in line with our existing defaults.